client/header/activity-panel/panels/orders.js

```diff
@@ -64,49 +63,68 @@ function OrdersPanel( { orders, isRequesting, isError } ) {
 		</EllipsisMenu>
 	);
 
-	const orderCardTitle = ( order, address ) => {
-		const name = `${ address.first_name } ${ address.last_name }`;
+	const getCustomerString = order => {
+		const { first_name, last_name } = order.extended_info.customer;
```

Comment:
While I generally trust our API, the output of it is fully filterable... as such do you think we need to verify `order.extended_info` exists first? Perhaps assign it to a const as it is used below too `const orderInfo = order.extended_info || {}`

Reply:
That's a good point.  Updated this in https://github.com/woocommerce/wc-admin/pull/1578/commits/46ad004045ba667e1084a900ec0e091b04e79cb5.

Side note: as it stands right now, quite a few of our other views would fail if `extended_info` had been filtered out.  We might want to consider fixing these across the board.

Reply:
Issue noted for other reports [here](https://github.com/woocommerce/wc-admin/issues/1595).